code/obj/item/clothing/uniforms.dm

```diff
@@ -321,6 +321,19 @@
 		icon_state = "hop-dress"
 		item_state = "hop-dress"
 
+	fancy/bland
+		icon_state = "hop-fancy-alt"
+		item_state = "hop-fancy-alt"
+
+	dress/bland
```

Comment:
You have `dress/bland` twice here

Reply:
thank you for that, i was wondering why it didnt show up on the admin spawn list